lib/sinon.js

```diff
@@ -1,42 +1,64 @@
+var extend = require("./sinon/util/core/extend");
+var fakeTimers = require("./sinon/util/fake_timers");
+var format = require("./sinon/util/core/format");
+var nise = require("nise");
 var Sandbox = require("./sinon/sandbox");
-exports.sandbox = new Sandbox();
-exports.createSandbox = require("./sinon/create-sandbox");
+var stub = require("./sinon/stub");
 
-exports.expectation = require("./sinon/mock-expectation");
-exports.createStubInstance = require("./sinon/stub").createStubInstance;
+var apiMethods = {
+    createSandbox: createSandbox,
+    assert: require("./sinon/assert"),
+    match: require("./sinon/match"),
+    spy: require("./sinon/spy"),
+    spyCall: require("./sinon/call"),
+    stub: require("./sinon/stub"),
+    mock: require("./sinon/mock"),
 
-exports.defaultConfig = require("./sinon/util/core/default-config");
+    expectation: require("./sinon/mock-expectation"),
+    createStubInstance: require("./sinon/stub").createStubInstance,
+    defaultConfig: require("./sinon/util/core/default-config"),
 
-var fakeTimers = require("./sinon/util/fake_timers");
-exports.useFakeTimers = fakeTimers.useFakeTimers;
-exports.clock = fakeTimers.clock;
-exports.timers = fakeTimers.timers;
+    setFormatter: format.setFormatter,
 
-var nise = require("nise");
-exports.xhr = nise.fakeXhr.xhr;
-exports.FakeXMLHttpRequest = nise.fakeXhr.FakeXMLHttpRequest;
-exports.useFakeXMLHttpRequest = nise.fakeXhr.useFakeXMLHttpRequest;
+    // fake timers
+    useFakeTimers: fakeTimers.useFakeTimers,
+    clock: fakeTimers.clock,
+    timers: fakeTimers.timers,
 
-exports.fakeServer = nise.fakeServer;
-exports.fakeServerWithClock = nise.fakeServerWithClock;
 
-exports.createFakeServer = nise.fakeServer.create.bind(nise.fakeServer);
-exports.createFakeServerWithClock = nise.fakeServerWithClock.create.bind(nise.fakeServerWithClock);
+    // fake XHR
+    xhr: nise.fakeXhr.xhr,
+    FakeXMLHttpRequest: nise.fakeXhr.FakeXMLHttpRequest,
+    useFakeXMLHttpRequest: nise.fakeXhr.useFakeXMLHttpRequest,
 
-var behavior = require("./sinon/behavior");
+    // fake server
+    fakeServer: nise.fakeServer,
+    fakeServerWithClock: nise.fakeServerWithClock,
+    createFakeServer: nise.fakeServer.create.bind(nise.fakeServer),
+    createFakeServerWithClock: nise.fakeServerWithClock.create.bind(nise.fakeServerWithClock),
 
-exports.addBehavior = function (name, fn) {
-    behavior.addBehavior(exports.stub, name, fn);
+    addBehavior: function (name, fn) {
+        behavior.addBehavior(stub, name, fn);
+    }
 };
 
-var format = require("./sinon/util/core/format");
-exports.setFormatter = format.setFormatter;
+var legacySandboxAPI = {
+    sandbox: {
+        create: deprecated.wrap(
+            createSandbox,
+            // eslint-disable-next-line max-len
+            "`sandbox.create()` is deprecated. Use default sandbox at `sinon.sandbox` or create new sandboxes with `sinon.createSandbox()` or `new sinon.Sandbox()`"
```

Comment:
>  create new sandboxes with `sinon.createSandbox()` or `new sinon.Sandbox()
Why have two ways of creating sandboxes? We just made a new factory function.

Reply:
I'll remove the second way and tidy up the documentation. Thanks